### PR TITLE
Fixed the tests

### PR DESCRIPTION
**Pull request by rpgeeganage**
It seems Assert Error messages are different in different environments and `console.log` an object is different.
I run the tests in Node 8, 10, 12 environments.

### CODE REVIEW

File: test/tests.js [hunk 1]
```diff
@@ -1,51 +1,51 @@
...
-  }
-
-  assert.strictEqual(op, getSnapshotContent('snapshot/test_6.txt'));
-});
\ No newline at end of file
+runTest("test_case/test_6.js", "snapshot/test_6.txt", (err, op) => {
+    if (err) {
+        throw err;
+    }
+    const regex = /[\n\t\s\+']/g;
```
Comment (fatso83) [inline test/tests.js:49]: Yeah, this is the hoops one needs to go through when doing snapshot testing :-D

**Comment by fatso83** [comment]
Another way of doing snapshot testing, instead of normalizing the output, is to have a stable target. I have made the required changes in [the Circle CI config](https://github.com/fatso83/mini-mocha/blob/master/.circleci/config.yml#L33), should we decide on doing this later.

**Comment by rpgeeganage** [comment]
Yeah. I’ll look into stabilizing the snapshot testing. It’s nice to have CI tool. 😁

**Comment by fatso83** [comment]
All tests passed! :boom:  Merging.

**Comment by fatso83** [comment]
@rpgeeganage Don't bother doing much now. It's _fine_. You _did_ stabilize it with this fix :)

**Pull request merged**